src/main/scala/treadle/executable/Memory.scala

```diff
@@ -163,12 +163,12 @@ object Memory {
       sensitivityGraphBuilder.addSensitivity(mask, valid)
 
       val pipelineReadDataSymbols = (0 until memory.readLatency).flatMap { n =>
```

Comment:
Should the name change to `pipelineReadAddrSymbols` if it's pipelining the address?

Reply:
Probably yeah. But this code is completely dead anyways since there is no test case that has a combined R/W port. Also: the firrtl passes that treadle currently uses split up all R/W ports and turn them into separate ports.

Addressing the whole issue of R/W ports is outside the scope of this issue, but yeah, this code should be tested+fixed or removed eventually.

Reply:
The issue that triggered this pull request was the first line I fixed. I only extended it to the other two lines for consistency's sake.